app/players/elves/elf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Elf(Player):
+    def __init__(self, nickname: str, musical_instrument: str) -> None:
+        super().__init__(nickname)
+        self.musical_instrument = musical_instrument
+
+    def play_elf_song(self) -> None:
+        print(f"{self.nickname} is playing a song on the {self.musical_instrument}")
```

Comment:
Update the `play_elf_song` method to use the protected attribute `_musical_instrument`.